rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,435 @@
+
+UX MUST review the proposed survey with the DCP community and incorporate suggestions as appropriate.
+
+The survey MUST be announced to the membership of the Human Cell Atlas (through its Registration mailing list) with a minimum of a two week last call for responses.
+
+At the conclusion, UX MUST summarize insights from the survey and present at the next DCP PM meeting. Following a one week review to incorporate feedback, the survey MUST be shared by UX with the membership of the Human Cell Atlas and announced on the *HumanCellAtlas #dcp* slack channel.
+
+UX MAY also share critical UX Insights from other sources independent of the Survey for consideration by the Project Leads during the creation of the Roadmap.
+
+#### Roadmap Retrospective
+
+The Roadmap Retrospective occurs during the recurring Project Leads meeting and not in a special session. The Roadmap Shepherd MUST add the Retrospective to the agenda and faciliate.
+
+In preparation, the Project Leads MUST independently review the prior year’s Product Roadmap, Release Retrospective summaries, and appropriate supporting documentation and then share their significant findings and learnings with their peers during the Retrospective.
+
+Findings may include: Themes that were incomplete and the accompanying reason, incorrectly sized work in the Engineering Plan, etc. These findings SHOULD be utilized as a reference class for the next Product Roadmap.
+
+Project Leads MUST identify a Roadmap Shepherd during the Retrospective. The Shepherd MUST NOT be a Project Lead, to ensure the neutrality of the content for the Roadmap.
+
+The Shepherd is responsible for:
+* Managing the timeline for the creation of the draft Roadmap
+* Facilitating all meetings related to the creation of the draft Roadmap 
+* Documenting important decisions, discussions, and questions 
+* Assigning action items to move toward the creation of the Roadmap, should no Project Lead volunteer, and following up with them through completion
+
+### Drafting the Roadmap
+[Drafting the Roadmap]:#drafting-the-roadmap
+
+Project Leads MUST independently create Themes by utilizing the input provided in the Science Roadmap, Technical Objectives RFC, DCP User Survey and UX Insights, and Roadmap Retrospective.
+
+To create the draft Product Roadmap, the Roadmap Shepherd:
+- Forks the *HumanCellAtlas dcp-community* repository
+- Copies `rfcs/roadmap-template.md` to `rfcs/text/0000-roadmap-YYYY.md` - where "YYYY" is the target year for the roadmap. 
+
+The Roadmap Shepherd MUST collate all themes into the draft which is an Informational RFC. 
+
+Project Leads MUST review the Themes as a collective and distill the revised list to a lesser number of themes. This is achieved through a consensus building process, which is implemented by the Roadmap Shepherd and occurs during specifically noted Project Lead meetings, outlined below:
```

Comment:
@briandoconnor - you also wrote:

> Finally, I think the process overall is heavy and complex. I feel like we lack forward momentum in HCA and I don't think more process is going to address that core issue. So thinking of ways to streamline this roadmapping and making it as lightweight as possible so folks can focus on work is important.

The aspiration was for the Project Leads to self-organize and  _"prototype_"  their own roadmap process which could then be documented by Cara acting as editor. Failing that, Cara wrote a proposal. Could you make concrete recommendations for a lighter and streamlined process to produce a multiple-quarter roadmap per the requirement from Oversight? What would you eliminate or simplify in the proposed roadmap process?

For example, the RFC could simply provide a template and state that the "Project Leads will provide an annual roadmap using the template by <date> based on a process of their own choosing". If the time box was exceeded, then the matter is escalated to Oversight. This would be very light process but I suspect that the Project Leads would continue to struggle.